src/Storage/Storage.Management/Models/PSDataPolicy.cs

```diff
@@ -382,29 +421,10 @@ public PSDateAfterCreation(int daysAfterCreationGreaterThan)
         {
             this.DaysAfterCreationGreaterThan = daysAfterCreationGreaterThan;
         }
-
         public PSDateAfterCreation(int daysAfterCreationGreaterThan, int? DaysAfterLastTierChangeGreaterThan)
         {
             this.DaysAfterCreationGreaterThan = daysAfterCreationGreaterThan;
             this.DaysAfterLastTierChangeGreaterThan = DaysAfterLastTierChangeGreaterThan;
         }
-
-        public PSDateAfterCreation(DateAfterCreation data)
-        {
-            this.DaysAfterCreationGreaterThan = Convert.ToInt32(data.DaysAfterCreationGreaterThan);
-            if (data.DaysAfterLastTierChangeGreaterThan is null)
-            {
-                this.DaysAfterLastTierChangeGreaterThan = null;
-            }
-            else
-            {
-                this.DaysAfterLastTierChangeGreaterThan = Convert.ToInt32(data.DaysAfterLastTierChangeGreaterThan);
-            }
-        }
-        public DateAfterCreation ParseDateAfterCreation()
-        {
-            return new DateAfterCreation(this.DaysAfterCreationGreaterThan, this.DaysAfterLastTierChangeGreaterThan);
```

Comment:
Same comment should be add to code for SDK still not support "DaysAfterLastTierChangeGreaterThan". Like // TODO: DaysAfterLastTierChangeGreaterThan is still not supported by SDK, will add later.

Reply:
Done